include/matx/operators/slice.h

```diff
@@ -250,14 +285,24 @@ namespace matx
    * @param ends the last element (exclusive) of each dimension of the input operator.  matxDrop Dim removes that dimension.  matxEnd deontes all remaining elements in that dimension.
    * @return sliced operator
    */
+  template <int N, typename OpType>
+  __MATX_INLINE__ auto slice (const OpType opIn, 
+      const std::array<index_t, OpType::Rank()> &starts,
+      const std::array<index_t, OpType::Rank()> &ends)
+  {
+    std::array<index_t, OpType::Rank()> strides;
+    for(int i = 0; i < OpType::Rank(); i++)
```

Comment:
`strides.fill`?

Reply:
Although. std::array isn't officially in libcudacxx until later versions.

Reply:
This didn't seem to impact compile or correctness.  I think these are on host only thus not an issue?

Reply:
Yeah, so can it be switched to fill to save a line?

Reply:
fixed